### DISCUSSION TITLE
Maybe it's time to rename this package from `react-native-tvos` to `react-native-tv`

### DISCUSSION THREAD

**Discussion by retyui** [react-native-tvos/react-native-tvos#845] [General Discussions]
The current name, react-native-tvos, suggests a focus solely on Apple's tvOS platform, potentially excluding other TV platforms that also utilize the library. As React Native continues to expand its reach and capabilities, the proposed name react-native-tv reflects a broader scope, accommodating all TV-based platforms, including tvOS, Android TV, FireTV, and other potential platforms.

Inclusivity: The new name is more inclusive of all TV platforms, promoting the library's relevance beyond tvOS.
Clarity: A simplified name aligns better with React Native's naming conventions (e.g., `react-native-web`, `react-native-windows`).

**Comment by dcordero** [comment]
This is an open discussion here (https://github.com/react-native-tvos/react-native-tvos/discussions/737)
